### ISSUE TITLE
[Elden Ring Inventory Tool] More accessible damage calculator

### ISSUE THREAD

**Issue by Owlvard** [sovietspaceship/souls-bug-reports#29]
**Application**

-   Elden Ring Build Planner / Inventory Tool (https://er-build-planner.nyasu.business)

**Is your feature request related to a problem? Please describe.**
I want to compare how much damage I lose/gain by changing my damage spread, like from 36/36 to 44/28
With 1h R1 attacks I can simply hover over the weapon and see the damage change there.
But with other attacks, like powerstance attacks this won't be accurate.

**Describe the solution you'd like**
Some way to set which attack is used for the damage tooltip when hovering over weapons

**Describe alternatives you've considered**
Maybe another view that lets you have the damage calculator visible and not reset in the overview window.
You'd select your weapon from the ones you added to your inventory, rest is same as the one available when adding a new weapon to your inventory

**Additional context**
![image](https://github.com/sovietspaceship/souls-bug-reports/assets/30782589/302dad19-bf2d-49c3-affd-d6458dfa8c95)
![image](https://github.com/sovietspaceship/souls-bug-reports/assets/30782589/73f724c4-4077-4d1e-b37c-026eb213e160)

**Comment by sovietspaceship** [comment]
good idea, (when I'm back from vacation) I'll make the damage calculator a standalone component instead of just sitting in the weapon slot edit screen, and maybe add an attack dropdown to the filters in the weapon picker so when sorting by damage it uses that attack
